components/raftstore/src/store/peer.rs

```diff
@@ -2343,6 +2394,8 @@ where
             "peer_id" => self.peer_id(),
         );
 
+        self.disable_apply_unpersisted_log(self.raft_group.raft.raft_log.last_index());
```

Comment:
If not leader, just disabling `apply_unpersisted_log` is enough. It seems that setting `min_safe_index_for_unpersisted_apply` is useless.

Reply:
Yes, I just reuse the same logic here since set its value has no other side effect. It can also slightly make it easier to handle once we want to also do it in follower.

Reply:
The side effect is that it may take longer to wait for the applied index to be greater than the last index when this peer was isolated and had lots of raft logs that could not be committed.

Reply:
got it.